### PR TITLE
Renamed hasSize(int, int) to hasDimensions(int, int) for two-dimensio…

### PR DESCRIPTION
**Pull request by mwajcht**
…nal array asserts

#### Check List:
* Fixes #1182 
* Unit tests : YES
* Javadoc with a code example (on API only) : YES

### CODE REVIEW

File: src/main/java/org/assertj/core/api/Array2DAssert.java [hunk 1]
```diff
@@ -79,20 +79,20 @@
    * <p>
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/api/Array2DAssert.java:79]: `Verifies that the number of values in the actual array is equal to the given one.`
should be reworded to:
`Verifies that the actual 2D array has the same dimensions as the given one.`

File: src/main/java/org/assertj/core/api/Array2DAssert.java [hunk 2]
```diff
@@ -79,20 +79,20 @@
...
    * </code></pre>
    *
    * @param expectedFirstDimensionSize the expected number of values in first dimension of the actual array.
    * @param expectedSecondDimensionSize the expected number of values in second dimension of the actual array.
    * @return {@code this} assertion object.
-   * @throws AssertionError if the number of values of the actual array is not equal to the given one.
+   * @throws AssertionError if the actual array's dimensions are not equal to the given ones.
    */
-  SELF hasSize(int expectedFirstDimensionSize, int expectedSecondDimensionSize);
+  SELF hasDimensions(int expectedFirstDimensionSize, int expectedSecondDimensionSize);
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/api/Array2DAssert.java:95]: rename:
* `expectedFirstDimensionSize` -> `expectedFirstDimension` 
* `expectedSecondDimensionSize` -> `expectedSecondDimension`

File: src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java [hunk 3]
```diff
@@ -24,7 +24,7 @@
 import org.junit.jupiter.api.Test;
 
 /**
- * Tests for <code>{@link Char2DArrays#assertHasSize(AssertionInfo, char[][], int, int)}</code>.
+ * Tests for <code>{@link Char2DArrays#assertHasDimensions(AssertionInfo, char[][], int, int)}</code>.
  *
```
Comment (joel-costigliola) [inline src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java:28]: the class name should be renamed to `Char2DArrays_assertHasDimensions_Test`

Comment (joel-costigliola) [reply src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java:28]: the same applies for other tests

File: src/main/java/org/assertj/core/api/Array2DAssert.java (hunk 1 above)
Comment (mwajcht) [reply src/main/java/org/assertj/core/api/Array2DAssert.java:79]: done

File: src/main/java/org/assertj/core/api/Array2DAssert.java (hunk 2 above)
Comment (mwajcht) [reply src/main/java/org/assertj/core/api/Array2DAssert.java]: done

File: src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java (hunk 3 above)
Comment (mwajcht) [reply src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java:28]: done

**Comment by joel-costigliola** [comment]
Integrated thanks @mwajcht!

**Pull request closed without merging**